spec.md

```diff
@@ -2255,7 +2257,9 @@ value in the range -`n` ≤ `i` < `n`, where `n` is `len(a)`; any other
 index results in an error.
 
 ```text
-SliceSuffix = '[' [Expression] [':' Test [':' Test]] ']' .
+SliceSuffix = '[' [Expression] ':' [Test] [':' [Test]] ']'
+            = '[' Expression ']'
```

Comment:
The '=' should be a '|'.

Though perhaps it would be clearer to say:

```
SliceSuffix = '[' Expression ']'.
```
or
```
SliceSuffix = '[' Expression ']'.
                   | ...
```

This avoids the repetition of irrelevant productions, though it does raise the question of what "..." means in this formal grammar.

Reply:
Typo. Thanks!